app/services/person_with_remote_fields.rb

```diff
@@ -0,0 +1,49 @@
+class PersonWithRemoteFields < SimpleDelegator
+  REMOTE_FIELDS = [
+    :email, :phone, :first_name, :last_name, :is_volunteer,
+    :employer, :occupation, :skills, :tags
+  ]
+  LOCAL_FIELDS = [
+    :email, :phone, :first_name, :last_name,
+    :address, :city, :zip, :is_volunteer
+  ]
+
+  def self.find_or_build(attributes)
+    email = attributes.symbolize_keys.delete(:email)
+    person = Person.find_or_initialize_by(email: email)
```

Comment:
I know you want to refactor this later, but we need to make sure we use the find by email, uuid or phone prior to going to production with this.